src/components/anonymous-modal.jsx

```diff
@@ -6,7 +6,7 @@ import CurrentUserStore from '../user-store';
 
 let hasAlreadyShownAnonymousModal = false;
 
-const AnonymousModal = React.createClass({
+class AnonymousModal extends React.Component {
```

Comment:
Could just be me, but I much prefer `extends Component` over `extends React.Component`. (i.e. importing `Component` using `import React, {Component} from 'react'`)

Reply:
Yeah, let's do that.

Reply:
Additionally, [babel-plugin-react-require](https://www.npmjs.com/package/babel-plugin-react-require) can be used to auto import React when the file only has functional component that doesn't need to extends from `Component`